### ISSUE TITLE
[A11Y][Password Input] Change the accessible name on show/hide button

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#599]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

S25

### OS Version

16

### App version

1.0.0

### Describe the issue

The button to show/hide the password is named with the current status and the hint, for example, "Password hidden, double tap to show password"

It works, but it would be easier to name the button with the next status

### Expected Behavior

- if password is hidden, then buttton should be named "Show password"
- if password is visible, then buttton should be named "Hide password"
- No hint needed anymore

**Comment by pya35** [comment]
OK
